alerts/google-network-intelligence-center/packet-loss-high.v1.json

```diff
@@ -0,0 +1,25 @@
+{
+  "displayName": "High Packet Loss Alert Policy",
+  "documentation": {
+    "content": "This alert indicates that the Packet Loss exceeded 5% for 5 minutes for a specific region pair.",
+    "mimeType": "text/markdown"
+  },
+  "userLabels": {},
+  "conditions": [
+    {
+      "displayName": "Packet Loss higher than 5%",
+      "conditionMonitoringQueryLanguage": {
```

Comment:
Please note that MQL alert policies are not yet supported by the in-context alerting component.

Reply:
There are already 30 sample alerts using MQL - https://source.corp.google.com/search?q=conditionMonitoringQueryLanguage%20f:third_party%2Fcloud_monitoring_sample_dashboards.*json&sq= - how come?